bot-commands.md

```diff
@@ -0,0 +1,37 @@
+| !discord      | Feel free to join our Discord to continue the discussion: https://red.ht/discord |
+| !ebooks       | Need some reading material on some cloud native software? We've got just the thing! Check out all our free eBooks: https://red.ht/all-the-ebooks |
+| !helpernode   | In this live stream, Christian Hernandez and Chris Short walk through what helpernode is, how to get the helpernode up and running, and getting an OpenShift 4 cluster stood up with helpernode's assistance. https://youtu.be/wZYx4_xBSUQ |
+| !knative      | Get a free copy of the O'Reilly Knative Cookbook by Burr Sutter & Kamesh Sampath https://red.ht/knative-ecookbook |
+| !learn       | Checkout https://red.ht/stream-learn for Katacoda courses designed to help you learn more about Red Hat OpenShift |
+| !makeyourmark | Join a community of passionate tech users, customers, and practitioners The Red Hat Accelerators program is a global enterprise customer community of Red Hat technology experts and enthusiasts who willingly share their knowledge and expertise with peers in the industry, their community, and with Red Hat. https://red.ht/makeyourmark |
+| !operators   | Get the Kubernetes Operators O'Reilly book for free! This book explains what an Operator is and how Operators extend the Kubernetes API. It shows how to deploy and use existing Operators, and how to create and distribute Operators for your applications using the Red Hat Operator Framework. https://red.ht/operators-ebook |
+| !patterns     | Get your own copy of the Kubernetes Patterns O'Reilly eBook for free! With this book, you'll learn to create cloud-native applications with Kubernetes as a runtime platform, manage container and platform interactions, apply configuration patterns to your Kubernetes-based applications, and build container images directly within the cluster. https://red.ht/k8s-patterns-ebook |
+| !slides       | If slides are available they are posted at https://speakerdeck.com/redhatopenshift |
+| !tluh        | Want to take the next step in leveling up your knowledge and career? We’ve got you covered! Head over to https://red.ht/leveluphour to take advantage of learning materials and exam discounts. |
+| !try         | Get started with OpenShift — Try Red Hat OpenShift 4 now, or set up your own clusters in the environment of your choice. https://red.ht/stream-try |
+| !welcome     | Welcome all! Thank you for joining us. We're happy to have you here with us today. Feel free to ask questions and we'll be happy to answer. Enjoy! |
+
+## Streamlabs
+
+The following bot commands are for use in Twitch chat and are controlled by [Streamlabs](https://streamlabs.com/dashboard#/cloudbot/commands/custom):
+
+| Command | Output |
+|:-------------|:-------|
+| !followage   | {touser.name} has been following {channel.name} for {touser.followage}. |
+| !hug         | /me hugs {target.name} <3 |
```

Comment:
Do these two commands require any other inputs? It's not clear to me how `!followage` would actually know what user or channel...

Reply:
Everything inside the curly brackets are variables that Streamlabs uses.

These were "suggested" commands. We've never seen anyone use them though. I'm indifferent on keeping them.